apps/ui/components/reservation-unit/QuickReservation.tsx

```diff
@@ -431,7 +421,7 @@ const QuickReservation = ({
     formatDate(nextHour.toISOString(), "HH:mm")
   );
   const [duration, setDuration] = useState<OptionType | undefined>(
-    durationOptions.find((n) => n.value === "1:00") || durationOptions[0]
+    durationOptions.find((n) => n.value === "60 min") ?? durationOptions[0]
   );
```

Comment:
Value is never `60 mins`. Using a number for the value is the correct solution here.

Reply:
So it was incorrect to start with (as it was using a string in different format)?

Reply:
It should use the "1:00" which was there to begin with, methinks. (as the option's `value` is still `1:00` - only labels changed)

Reply:
...altho I wonder if that entire comparison is actually unnecessary. What we want is `durationOptions[0]` anyway - don't know why try to force the 1h value there if the reservation unit has a different minimum reservation duration.

Reply:
Asked Krista, and we should go with the 1h assumed default time (and not the first option, unless there isn't a valid 1h option).